src/ORM/Rule/ExistsIn.php

```diff
@@ -40,14 +40,28 @@ class ExistsIn
     protected $_repository;
 
     /**
+     * Options for the constructor
+     *
+     * @var array
+     */
+    protected $_options = [];
```

Comment:
It applies all the warnings and errors from the PHPCS rules. I think we've generally silenced warnings in our travis outputs.